entry_test.go

```diff
@@ -43,17 +50,234 @@ func TestEntryPanicln(t *testing.T) {
 		case *Entry:
 			assert.Equal(t, "kaboom", pVal.Message)
 			assert.Equal(t, errBoom, pVal.Data["err"])
+			assert.Regexp(t, `^level=panic msg=kaboom err="boom time"`, outBuffer.String())
```

Comment:
The key order depends on the implementation and is not guaranteed, this is why a json formatter if way more convenient to check the output of a log.